src/context/note-context.jsx

```diff
@@ -13,11 +13,12 @@ const NoteContextProvider = ({ children }) => {
     color: '#fff',
     archive: false,
     pinned: false,
+    tags: '',
   })
 
   const [notes, setNotes] = useState([])
   const [archiveNotes, setArchiveNotes] = useState([])
-
+  const [trash, setTrash] = useState([])
   return (
     <noteContext.Provider
```

Comment:
you can use camelcasing ( i.e first letter capital as well) for naming contexts as we usually use smallcase for html elements in react.

Reply:
will do it